ci/docker-compose-build.yaml

```diff
@@ -29,7 +29,7 @@ services:
     image: avalon-lmdb:${ISOLATION_ID}
     build:
       context: ..
-      dockerfile: examples/shared_kv_storage/Dockerfile
+      dockerfile: shared_kv_storage/Dockerfile
```

Comment:
This change is already present in latest code in master branch. Looks like you might need to rebase.

Reply:
this is very strange, I rebased but it still shows this.

Reply:
The commits in the PR shows other commits as well. Probably, you included those commits while rebasing. that's why you see the above changes in this PR.

Reply:
The commit on my own remote repo shows just one commit. But here shows 3. Is there an easy way to fix this not losing all the comments of this PR? I mean not closing this PR.